Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/Util/DatabaseHelper.swift

```diff
@@ -714,6 +714,16 @@ class DatabaseHelper {
             }
         }
 
+        if schemaVersion < 48 {
```

Comment:
This is the same schemaVersion as the other PR, once one is merged I'll put both changes into the same schemaVersion.